R/run.R

```diff
@@ -320,21 +331,46 @@ view_run_details <- function(run) {
                     "Web View",
                     run$id,
                     status,
-                    format(parsedate::parse_iso_8601(details$startTimeUtc),
-                           format = "%B %d %Y %H:%M:%S"),
+                    start_time,
                     duration,
                     details$runDefinition$target,
                     details$runDefinition$script,
                     toString(details$runDefinition$arguments),
-                    web_view_link),
+                    link_warn),
                nrow = 8,
                ncol = 2)
 
   DT::datatable(df, escape = FALSE, rownames = FALSE, colnames = c(" ", " "),
-                caption = paste(unlist(details$warnings), collapse = "\r\n"),
+                caption = "Run Details",
                 options = list(dom = "t", scrollY = TRUE))
 }
 
+#' Show run details in viewer pane or browser
+#' @param run Run object
+#' @export
+view_run_details <- function(run) {
+  rstudio_server <- grepl("rstudio-server", Sys.getenv("RS_RPOSTBACK_PATH"))
+  
+  # Run widget unless in Notebook VM
+  if (!rstudio_server && rstudioapi::isAvailable()) {
```

Comment:
The upcoming notebook VM image release will have shinyapp issue solved so we can remove the check here